lib/Model.php

```diff
@@ -234,7 +235,7 @@ class Model
 	 * Constructs a model.
 	 *
 	 * When a user instantiates a new object (e.g.: it was not ActiveRecord that instantiated via a find)
-	 * then @var $attributes will be mapped according to the schema's defaults. Otherwise, the given
+	 * then <code>$attributes</code> be mapped according to the schema's defaults. Otherwise, the given
```

Comment:
Yea, it's just awkwardly incomplete to fix it once like this in the sentence and ignore the others. If you're going to change only once instance, I'd suggest just removing the `<code>` bits and leave it consistent with the rest.